### PR TITLE
v59.0.0

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by yarneo** [comment]
lemme know if you want me to review this also (not  due to codeowners), I verified the CLA

**Comment by jverkoey** [comment]
Please review the CHANGELOG description for accuracy and grammar.

**Review by yarneo — APPROVED** [review]
The CHANGELOG looks great 👍

File: CHANGELOG.md
Comment (romoore) [inline CHANGELOG.md:3]: Super-nit: The hyperlink should be around the descriptive text. That way a screen reader jumping between links can easily determine the entire context of the link.

Suggestion,
```
This major release [removed the remaining encoding/decoding behaviors from components (tracking project](https://github.com/material-components/material-components-ios/projects/22)) and fixed a variety of bugs in FlexibleHeader with relation to safe area insets.
```

File: CHANGELOG.md
Comment (romoore) [inline CHANGELOG.md:11]: Is the new behavior a class or instance property? Suggest using `- inferTopSafe...` or `+ inferTopSafe...` to make it clearer. I'm guessing from the instructions below that it's a class property? 

> To enable the new behavior, you simply set `inferTopSafeAreaInsetFromViewController` on `MDCFlexibleHeaderViewController` to `YES`.

File: CHANGELOG.md
Comment (romoore) [inline CHANGELOG.md:55]: Do we want to alphabetize these components?

**Review by romoore — APPROVED** [review]
Reviewed CHANGELOG. Please note if any code changes have been made for this release.

**Comment by romoore** [comment]
@jverkoey  You didn't like the suggestions?

**Comment by jverkoey** [comment]
Oh! I didn't see the email here because I'd already started the merge/deploy once I got the LGTM. Sorry @romoore that was an accident :(

**Comment by jverkoey** [comment]
Can retroactively fix the text though.

File: CHANGELOG.md
Comment (jverkoey) [reply CHANGELOG.md:3]: Done.

File: CHANGELOG.md
Comment (jverkoey) [reply CHANGELOG.md:11]: Done.

File: CHANGELOG.md
Comment (jverkoey) [reply CHANGELOG.md:55]: This part of the content is auto-generated.

**Comment by jverkoey** [comment]
https://github.com/material-components/material-components-ios/pull/4636

File: CHANGELOG.md
Comment (romoore) [reply CHANGELOG.md:55]: Sure, but I recall in the release instructions that we're encouraged to clean it up or fix-up any nits.

Comment (romoore) [reply CHANGELOG.md:55]: Or we could fix the release script to alphabetize the components sections.

Comment (jverkoey) [reply CHANGELOG.md:55]: > we could fix the release script to alphabetize the components sections.

+1!

**Pull request merged**
